packages/formation-react/src/components/EmergencyBanner/EmergencyBanner.stories.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import EmergencyBanner from './EmergencyBanner';
+
+export default {
+  title: 'Library/EmergencyBanner',
+  component: EmergencyBanner,
+  argTypes: {
+    visible: { control: 'boolean' },
+    title: { control: 'text' },
+    content: { control: 'text' },
+  },
+};
+
+const Template = (args) => <EmergencyBanner {...args} />;
+
+const defaultArgs = {
+  visible: true,
+  title: 'Coronavirus',
+  content: `<div>
+      <p>
+        For questions about COVID-19 and how it affects VA health care and
+        benefit services, visit our
+        <a href="/coronavirus-veteran-frequently-asked-questions/">
+          coronavirus FAQs
+        </a>
+        or read
+        <a href="https://www.publichealth.va.gov/n-coronavirus/">
+          VA’s public health response
+        </a>
+        .
+      </p>
+      <p>
+        Please contact us first before going to any
+        <a href="/find-locations">VA location</a>. Contacting us first helps us
+        keep you safe.
+      </p>
+      <p>
+        For the latest coronavirus information, visit the
+        <a href="https://www.cdc.gov/coronavirus/2019-ncov/index.html">
+          CDC website
+        </a>
+        .
+      </p>
+    </div>`,
+};
+
+export const Default = Template.bind({});
+
+Default.args = {
+  ...defaultArgs,
+};
```

Comment:
```suggestion
};

export const ErrorBanner = Template.bind({});
ErrorBanner.args = { ...defaultArgs, type: 'error' };
```

Looks like there's some added functionality if `type === 'error'`. Probably could even add this as a `'select'` in the `argTypes` up top, where the options are `undefined` and `'error'`.

Reply:
good catch! i'll update to add select functionality for type `error` and `undefined` .. thanks!